### PR TITLE
New case of resize_short_overlay

### PR DESCRIPTION
**Pull request by hellohellenmao**
Check the data after resizing short overlay over longer backing files.

ID: 1811588
Signed-off-by: Tingting Mao <timao@redhat.com>

### CODE REVIEW

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.resize_short_overlay: PASS (4.37 s)

**Comment by hellohellenmao** [comment]
@lolyu @YongxueHong Help to take a review, please.

**Comment by lolyu** [comment]
test result, please.

**Comment by hellohellenmao** [comment]
@lolyu Help to merge if it's okay to you, pls.

File: qemu/tests/resize_short_overlay.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    root_dir = data_dir.get_data_dir()
+    base = QemuImg(params.object_params(images[0]), root_dir, images[0])
+    mid = QemuImg(params.object_params(images[1]), root_dir, images[1])
+    top = QemuImg(params.object_params(images[-1]), root_dir, images[-1])
+
+    logging.info("Create base and snapshot files")
+    for image in (base, mid, top):
+        image.create(image.params)
+
+    base_cmd = "write -P1 0 %s" % params["image_size_base"]
```
Comment (lolyu) [inline qemu/tests/resize_short_overlay.py]: better to put those `qemu-io` related cmds into configs.

File: qemu/tests/cfg/resize_short_overlay.cfg [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    image_size_base = 2M
+    image_format_base = qcow2
+    image_name_mid = "images/mid"
+    image_size_mid = 1M
+    image_format_mid = qcow2
+    image_name_top = "images/top"
+    image_size_top = 2M
+    image_format_top = qcow2
+    # 2M - 1M ($image_size_top - $image_size_mid)
+    diff_size = 1M
```
Comment (lolyu) [inline qemu/tests/cfg/resize_short_overlay.cfg]:
```
base_cmd = "write -P1 0 ${image_size_base}"
```

File: qemu/tests/resize_short_overlay.py (hunk 1 above)
Comment (hellohellenmao) [reply qemu/tests/resize_short_overlay.py]: Updated, thanks.

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.resize_short_overlay:PASS

**Comment by hellohellenmao** [comment]
@lolyu Review again, please~

**Pull request merged**